components/create-collective/NewCreateCollective.js

```diff
@@ -71,7 +71,7 @@ class NewCreateCollective extends Component {
     } else if (query.category === 'climate') {
       this.setState({ category: 'climate' });
     } else if (!query.category) {
-      this.setState({ category: null });
+      this.setState({ category: null, tos: null });
```

Comment:
Can't we just have `tos: null`  as a default value in the state constructor?

Reply:
Yes - updated